src/integrations/persona.ts

```diff
@@ -40,13 +40,27 @@ export function triggerPersona(
     fields: personaPrefill,
     onCancel: () => setErr('The verification was cancelled'),
     onError: (error: string) => setErr(`Verification error: ${error}`),
-    onComplete: ({ status }: any) => {
+    onComplete: async ({ status }: any) => {
       if (statusKey) {
         const submitStatus = { [statusKey]: status };
         updateFieldValues(submitStatus);
         featheryClient.submitCustom(submitStatus, { shouldFlush: true });
       }
-      onComplete();
+
+      const result = await featheryClient.pollPersonaResponse();
+      if (result?.status === 'complete') {
+        console.log('Persona polling complete:', result.value);
+        const submitStatus = { [statusKey]: result.value };
+        updateFieldValues(submitStatus);
+        featheryClient.submitCustom(submitStatus, {
```

Comment:
this isn't necessary, you just pulled this from the BE and dont need to send it back to BE